test/unit/cute/intel_xe/gemm_layout_data_type.cpp

```diff
@@ -0,0 +1,217 @@
+/***************************************************************************************************
+ * Copyright (c) 2024 - 2024 Codeplay Software Ltd. All rights reserved.
```

Comment:
@aacostadiaz should this be the codeplay header or should this be an intel one?

Reply:
@mehdi-goli do you know about this?

Reply:
Codeplay one is fine, year should be 2025